README.md

```diff
@@ -131,3 +131,9 @@ Due to the complexity of the `seek()` operation with large messages, users are e
 
 #### Log compacted topics ####
 In general, li-apache-kafka-clients does not support large message handling for log compacted topics. In some specific cases, it might work by playing with the message keys, users may refer to [Handle Large Messages in Apache Kafka](http://www.slideshare.net/JiangjieQin/handle-large-messages-in-apache-kafka-58692297) if interested.
+
+#### Record Headers ####
+With li-apache-kafka-clients_1.0.17 and onwards, we have introduced internal headers(with `_` as prefix) which are used to convey certain information about the messages. 
```

Comment:
It may not be super clear to the readers what "internal headers" exactly means here. You may want to provide a bit more info.